python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -1107,6 +1107,29 @@ def test_to_pandas_from_mixed_dataframe(self):
                 pdf_with_only_nulls = self.spark.sql(sql).filter("tinyint is null").toPandas()
                 self.assertTrue(np.all(pdf_with_only_nulls.dtypes == pdf_with_some_nulls.dtypes))
 
+    @unittest.skipIf(
+        not have_pandas or not have_pyarrow,
+        cast(str, pandas_requirement_message or pyarrow_requirement_message),
+    )
+    def test_to_pandas_for_array_of_struct(self):
```

Comment:
Could you mark to skip this test if the underlying PyArrow is less than `2.0.0`?